stdlib/public/runtime/ImageInspectionCOFF.cpp

```diff
@@ -54,8 +55,7 @@ void _swift_win32_withDbgHelpLibrary(
     }
 
     // If we have not previously initialized the Debug Help library, do so now.
-    bool isDbgHelpInitialized = false;
-    if (dbgHelpHandle) {
+    if (dbgHelpHandle && !isDbgHelpInitialized) {
```

Comment:
So, it's more complicated than this. We are _supposed_ to call `SymInitialize()` multiple times, but follow them up with calls to `SymCleanup()`. Which is probably what's missing.

(At one point, Swift Testing was going to hook into `_swift_win32_withDbgHelpLibrary()`, but it doesn't do the backtrace work we thought we needed, so it's become moot.)

Reply:
Why would we want to call `SymInitialize()` and `SymCleanup()` over and over? That's going to be expensive, I would imagine, as it's going to go off looking for symbols and things.

Reply:
Because this library is thread-unsafe to call in the general case, and at least in theory calling SymCleanup() means client code that calls into it is less likely to stomp.

Reply:
The documentation for `SymInitialize()` is unhelpfully vague, but my reading of it is that it should be OK to call `SymInitialize()` multiple times with _different_ process handles — that is, I think you can have multiple sessions active simultaneously. The API is still thread-unsafe, so needs mutual exclusion, but I don't think we need to `SymInitialize()` and `SymCleanup()` each time.

Reply:
It's admittedly been a while since I touched this code, and I've paged the dbghelp knowledge out of my head by now.